### DISCUSSION TITLE
修改前端页面样式不生效

### DISCUSSION THREAD

**Discussion by lainbo** [vvbbnn00/WARP-Clash-API#96] [Q&A]
**出现的问题**
修改`template/index.html`中的前端样式后, 重启docker, 前端页面依旧保持原样, 并没有我做的改动

**复现步骤**
1. 先git clone仓库, 配置.env.local, 其余什么都不改, 使用`docker-compose up -d`直接运行
2. 修改`template/index.html`
3. 使用`docker-compose down && docker-compose up -d`重启项目
4. 发现页面依旧是最开始clone下来的样子, 并没有第2步做的修改

**额外的补充**
还试验过完全删除文件夹, 重新clone等方法, 总之21001端口上的页面只会停留在第一次docker启动的时候, 后面的怎么改index.html都不会变

**Comment by vvbbnn00** [comment]
你好，这是因为您在修改页面后没有重新编译docker镜像导致的，您可以尝试`docker-compose up -d --build`来解决该问题。

**Comment by lainbo** [comment]
非常感谢~
